lib/less/tree/ruleset.js

```diff
@@ -2,6 +2,7 @@ var Node = require("./node"),
     Rule = require("./rule"),
     Selector = require("./selector"),
     Element = require("./element"),
+    Paren = require("./paren"),
```

Comment:
It is required [here](https://github.com/SomMeri/less-rhino.js/blob/20e555bf4749abd43a9a89337278ab3c54465d1d/lib/less/tree/ruleset.js#L482). Nested paths with replaced parent selectors need to be packed back into selector structure.

But, do not merge yet, I just realize the line I linked probably has bug (it does not referenced loop variable and probably should).